roxie/ccd/ccdserver.cpp

```diff
@@ -8285,15 +8281,9 @@ class CRoxieServerSortActivityFactory : public CRoxieServerActivityFactory
     {
         Owned<IHThorSortArg> sortHelper = (IHThorSortArg *) helperFactory();
         const char *algorithmName = NULL;
-        IHThorAlgorithm *sortMethod = static_cast<IHThorAlgorithm *>(sortHelper->selectInterface(TAIalgorithm_1));
-        if (sortMethod)
-        {
-            sortFlags = sortMethod->getAlgorithmFlags();
-            if (sortFlags & TAFconstant)
-                algorithmName = sortMethod->getAlgorithm();
-        }
-        else
-            sortFlags = TAFstable|TAFconstant;
+        sortFlags = sortHelper->getAlgorithmFlags();
+        if (sortFlags & TAFconstant)
```

Comment:
Can you think of any sensible reason why we should support dynamic algorithms?  I would be tempted to kill that feature.

Reply:
The only reason was in case it helped writing testing/timing code